### ISSUE TITLE
gzip:  zgrep specifies nonexistent GREP path

### ISSUE THREAD

**Issue by dredmorbius** [termux/termux-packages#1161]
**Problem:**  zgrep shell script specifies a nonexistant user's directory for GREP

    grep='${GREP-'\''/home/fornwall/lib/prefix/bin/grep'\''}'

Should be the system GREP utility:

    grep='${GREP-$PREFIX/bin/grep}'﻿

This may be an upstream issue -- it appears as if the alternative is a developer's own testbed.

Can I ping @fornwall?  I think so :)

**Comment by dredmorbius** [comment]
Package info:

```

Package: grep
Version: 3.1
Maintainer: Fredrik Fornwall @fornwall
Installed-Size: 221 kB
Depends: pcre
Homepage: https://www.gnu.org/software/grep/
Download-Size: 87.1 kB
APT-Manual-Installed: yes
APT-Sources: http://termux.net stable/main arm Packages
Description: Command which searches one or more input files for lines containing a match to a specified pattern
```

**Comment by fornwall** [comment]
Thanks for reporting! This has now been fixed in version `1.8-1` of the gzip package!
